internal/driver/gomobile/canvas.go

```diff
@@ -77,7 +77,7 @@ func (c *mobileCanvas) Refresh(obj fyne.CanvasObject) {
 }
 
 func (c *mobileCanvas) edgePadding() (topLeft, bottomRight fyne.Size) {
-	scale := fyne.CurrentDevice().SystemScale()
```

Comment:
I don't see the benefit of changing this file at all - since mobile isn't affected by the issues with SystemScale, and anyway mobile implementation calls SystemScaleForWindow(nil) underneath

Reply:
Because I didn't want to keep the "calling deperecated methods" in the codebase if I could avoid it :)

Reply:
Okay, go ahead